### ISSUE TITLE
When hero is currently activated via a click/tap, finish turn button does nothing

### ISSUE THREAD

**Issue by yonomitt** [abregado/broadstories#8]
When I have clicked on a hero to move him/her but decide not to move, I cannot simply click the finish turn button. I first have to deactivate the currently active hero and then the finish turn button functions.

**Comment by marc40000** [comment]
Also, if a unit is selected, clicking on another also does nothing. I expected that other unit to get selected.

**Comment by abregado** [comment]
Yes in the android version you can swap between units and end turn without deselecting your active unit. Perhaps I will move this functionality across to the Desktop versions.
